### DISCUSSION TITLE
Keypoint names: according to what output format ?

### DISCUSSION THREAD

**Discussion by HabibaAbderrahim** [open-mmlab/mmpose#2860] [Q&A] [answered]
Hello , after the inference using this command: python demo/inferencer_demo.py tests/data/segbig/img.jpg --pose2d yoloxpose_s_8xb32-300e_coco-640.py --pose2d-weights yoloxpose_s_8xb32-300e_coco-640-56c79c1f_20230829.pth --pose3d human3d --vis-out-dir vis_results/human3d/3_person_4_segmentationt.jpg --device cpu
and printing the 17 keypoints as follow : 
![image](https://github.com/open-mmlab/mmpose/assets/48824050/e943bfae-2e85-45a8-827b-1850a45b4d71)
 I wanted to make sure what each point represent in order to properly map it ( i mean for example : 
nose: [-0.0, 0.0, 0.6314282417297363]
left_eye: [0.10871679335832596, 0.07088106125593185, 0.6354875564575195]
right_eye: [0.1409938931465149, -0.30314338207244873, 0.3504978120326996] ?
![image](https://github.com/open-mmlab/mmpose/assets/48824050/b03920a3-59b1-433c-bf64-f621eb209ad1)
![image](https://github.com/open-mmlab/mmpose/assets/48824050/1d12f634-9afe-4fb6-a5d7-90e6200e89a2)

**Comment by HabibaAbderrahim** [comment]
i just want to know the printed keypoints representing the Human3.6 keypoints in order like
 | 1     | Nose            |
| 2     | Neck            |
| 3     | Right Shoulder  |
| 4     | Right Elbow     |
| 5     | Right Wrist     |
or  representing the coco keypoints in order like 
1 & Nose 
2 & Left Eye 
3 & Right Eye 
4 & Left Ear 
 5 & Right Ear 

thank you ,

**Reply by Ben-Louis** [reply, marked as answer]
Hi @HabibaAbderrahim, the outputs of 3D models follow the keypoints definition in Human3.6M

**Reply by HabibaAbderrahim** [reply]
Thank you for your reply ,
